Package.swift

```diff
@@ -3,6 +3,9 @@
 
 import PackageDescription
 
+let version = "0.6.0"
```

Comment:
Will the version be set automatically?

Reply:
for now no, but easier to update the package. i was just trying to remove the code smell

Reply:
Yes, I liked both, this and the url string 👍